Tribler/community/tunnel/hidden_community.py

```diff
@@ -493,18 +548,23 @@ def on_intro_established(self, messages):
             self.request_cache.pop(u"establish-intro", message.payload.identifier)
             self._logger.info("Got intro-established from %s", message.candidate)
 
-    def create_rendezvous_point(self, hops, finished_callback):
+    def create_rendezvous_point(self, hops, finished_callback, info_hash):
         def callback(circuit):
             # We got a circuit, now let's create a rendezvous point
             circuit_id = circuit.circuit_id
+            self.infohash_rp_circuits[info_hash].append(circuit_id)
```

Comment:
See if we can get some consitstency in naming. I like infohash_rp_circuits, but then rename seeding_points to infohash_sp_circuits